django/contrib/admin/models.py

```diff
@@ -44,6 +44,27 @@ def log_action(
             change_message=change_message,
         )
 
+    def log_actions(self, user_id, queryset, action_flag, change_message=""):
+        """
+        Method to bulk_create multiple LogEntry objects
+        from a given queryset.
+        """
+        if isinstance(change_message, list):
+            change_message = json.dumps(change_message)
```

Comment:
I know this is copied from above but solely checking for a single type of iterable irks me :D

I wonder if we should improve this to `isinstance(change_message, (list, tuple))` which is a typical check done in other parts of Django.

Reply:
(felix or nessita will have the final say on this - do what you think is best then they'll either request a change or do it themselves 👍)